.jenkins/caffe2/build.sh

```diff
@@ -207,22 +194,33 @@ fi
 ###############################################################################
 # Configure and make
 ###############################################################################
-# Run cmake from ./build_caffe2 directory so it doesn't conflict with
-# standard PyTorch build directory. Eventually these won't need to
-# be separate.
-rm -rf build_caffe2
-mkdir build_caffe2
-cd ./build_caffe2
 
-# Configure
-${CMAKE_BINARY} "${ROOT_DIR}" ${CMAKE_ARGS[*]} "$@"
+if [[ -z "$INTEGRATED" ]]; then
+
+  # Run cmake from ./build_caffe2 directory so it doesn't conflict with
+  # standard PyTorch build directory. Eventually these won't need to
+  # be separate.
+  rm -rf build_caffe2
+  mkdir build_caffe2
+  cd ./build_caffe2
+
+  # Configure
+  ${CMAKE_BINARY} "${ROOT_DIR}" ${CMAKE_ARGS[*]} "$@"
+
+  # Build
+  if [ "$(uname)" == "Linux" ]; then
+    make "-j${MAX_JOBS}" install
+  else
+    echo "Don't know how to build on $(uname)"
+    exit 1
+  fi
 
-# Build
-if [ "$(uname)" == "Linux" ]; then
-  make "-j${MAX_JOBS}" install
 else
-  echo "Don't know how to build on $(uname)"
-  exit 1
+
+  sudo FULL_CAFFE2=1 python setup.py install
+  # TODO: I'm not sure why this is necessary
+  cp -r torch/lib/tmp_install $INSTALL_PREFIX
```

Comment:
Why is this needed? If setup.py doesn't work by itself then imo it doesn't work.

Reply:
Per https://github.com/pytorch/pytorch/pull/10327 this seems to be necessary. I can sign up to fix this later but I need this patch through to unblock tensor merge.